pages/docs/adapters/fenics/adapter-fenics.md

```diff
@@ -0,0 +1,25 @@
+---
+title: The FEniCS adapter
+permalink: adapter-fenics.html
+keywords: adapter, fenics
+summary: "A python package providing a preCICE-adapter for the open source computing platform FEniCS."
+---
+
+## FEniCS adapter
+
+For details on installation and usage of the adapter, please refer to its github repository [`precice/fenics-adapter`](https://github.com/precice/fenics-adapter).
```

Comment:
Here, a section in **Installation** and **Testing** should be provided.

Reply:
What about the **Configuration** section?